akismet-privacy-policies.php

```diff
@@ -278,84 +339,112 @@ public function get_settings_page() {
+									<?php foreach( $this->languages as $lang ) { ?>
+									<option value="<?php echo $lang ?>" <?php selected( $this->translation, $lang ) ?>><?php echo strtoupper( substr( $lang, 0, 2 ) ) ?></option>
+									<?php } ?>
+								</select>
+							</td>
+						</tr>
+						<tr valign="top">
+							<th scope="row"><label for="akismet_privacy_checkbox"><?php _e( 'Consent via checkbox', 'akismet-privacy-policies' ) ?></label></th>
+							<td>
+								<input type="checkbox" id="akismet_privacy_checkbox" name="akismet_privacy_notice_settings_<?php echo $this->translation ?>[checkbox]" value="1"
+									<?php if ( isset( $this->options[ 'checkbox' ] ) ) {
+										checked( '1', $this->options[ 'checkbox' ] );
+									} ?> />
+							</td>
+						</tr>
+						<tr valign="top">
+							<th scope="row"><label for="akismet_privacy_notice"><?php _e( 'Privacy Notice', 'akismet-privacy-policies' ) ?></label></th>
+							<td>
+								<textarea id="akismet_privacy_notice" name="akismet_privacy_notice_settings_<?php echo $this->translation ?>[notice]" cols="80" rows="10"
+								aria-required="true"><?php if ( isset( $this->options[ 'notice' ] ) ) {
+									if ( $this->translation != 'en_US' ) {
+										$msg = $this->mo->translate( $this->options[ 'notice' ] );
+									} else {
+										$msg = $this->options[ 'notice' ];
+									}
+									echo $msg;
 								} ?></textarea>
-							<br /><strong>Hinweis:</strong> HTML möglich
-							<br /><strong>Achtung:</strong> Im Hinweistext musst du manuell den Link zu deiner Datenschutzerklärung einfügen. Einen Mustertext für die Datenschutzerklärung findest du im Reiter "Hilfe", rechts oben auf dieser Seite.
-
-							<br /><strong>Beispiel:</strong> <?php echo esc_html( $this->notice ); ?>
-						</td>
-					</tr>
-					<tr valign="top">
-						<th scope="row"><label for="akismet_privacy_error_message">Fehler-Hinweis</label></th>
-						<td>
-							<textarea id="akismet_privacy_error_message" name="akismet_privacy_notice_settings[error_message]" cols="80"
-								rows="10" aria-required="true"><?php if ( isset( $options[ 'error_message' ] ) ) {
-									echo $options[ 'error_message' ];
+								<br /><?php _e( '<strong>Note:</strong> HTML is possible', 'akismet-privacy-policies' ) ?>
+								<br /><?php _e( '<strong>Attention:</strong> You will have to add the link to your privacy statement manually. In Wordpress 5 and later you can find a link to a guide for creating your own privacy statement under \'Settings\' &rarr; \'Privacy\'.', "akismet-privacy-policies" ) ?>
+								<br /><strong><?php _e( 'Example:', 'akismet-privacy-policies' ) ?></strong> <?php
+								if ( $this->translation != 'en_US' ) {
+									$example_notice = $this->mo->translate( $this->notice );
+								} else {
+									$example_notice = $this->notice;
+								}
+								echo esc_html( $example_notice );
```

Comment:
Please use `esc_html_e()` to echo and sanitize the output.

Reply:
Yes, better. Thanks.